content/getstarted/02-item.md

```diff
@@ -1,56 +1,241 @@
+serviceaccount/cdi-apiserver created
+rolebinding.rbac.authorization.k8s.io/cdi-apiserver created
+role.rbac.authorization.k8s.io/cdi-apiserver created
+rolebinding.rbac.authorization.k8s.io/cdi-extension-apiserver-authentication created
+role.rbac.authorization.k8s.io/cdi-extension-apiserver-authentication created
+service/cdi-api created
+deployment.apps/cdi-apiserver created
+serviceaccount/cdi-sa created
+deployment.apps/cdi-deployment created
+service/cdi-uploadproxy created
+deployment.apps/cdi-uploadproxy created
+$
+```
+
+At this point, we have a fully functioning OpenShift cluster with KubeVirt alongside Ember-CSI backed by a Ceph backend. We have also created a namespace/project called `sample-project` which we will use to deploy a VM using KubeVirt. Before we proceed further, source the `env.sh` file which helps create useful aliases for us to use.
 
 ```shell
-make cluster-up
-make cluster-sync
+$ source tools/env.sh
+$ k create -f examples/cirros-pvc.yml
+persistentvolumeclaim/cirros-pvc created
+$ k get pvc
+NAME         STATUS    VOLUME                 CAPACITY   ACCESS MODES   STORAGECLASS                    AGE
+cirros-pvc   Bound     pvc-b19271605d2611e9   3Gi        RWO            io.ember-csi.external-ceph-sc   3s
+$
+```
+
 
-# Modify the cluster for CSI
-csi/csi_up.sh
+If the `status` above says `BOUND`, we will then check to see if the CDI Importer pod has finished importing. We can tell that the import operation has been completed because the pod (job) vanishes on completion, so we'll only see the imported pod for a minute or so. When the pod is no longer alive we can proceed to create the VM.
 
-# Setup RBAC
-cluster/kubectl.sh create -f csi/rbac.yml
+```shell
+$ k get pod
+NAME                            READY     STATUS    RESTARTS   AGE
+pod/importer-cirros-pvc-8lr4k   1/1       Running   0          36s
+$ k logs pod/importer-cirros-pvc-8lr4k
+I0412 15:30:01.911266       1 importer.go:45] Starting importer
+I0412 15:30:01.920943       1 importer.go:58] begin import process
+I0412 15:30:01.920993       1 importer.go:82] begin import process
+I0412 15:30:01.921027       1 dataStream.go:293] copying "https://download.cirros-cloud.net/0.4.0/cirros-0.4.0-x86_64-disk.img" to "/data/disk.img"...
+I0412 15:30:02.407774       1 prlimit.go:107] ExecWithLimits qemu-img, [info --output=json https://download.cirros-cloud.net/0.4.0/cirros-0.4.0-x86_64-disk.img]
+I0412 15:30:03.310515       1 prlimit.go:107] ExecWithLimits qemu-img, [convert -p -f qcow2 -O raw json: {"file.driver": "https", "file.url": "https://download.cirros-cloud.net/0.4.0/cirros-0.4.0-x86_64-disk.img", "file.timeout": 3600} /data/disk.img]
+I0412 15:30:03.320607       1 qemu.go:189] 0.00
+I0412 15:30:04.207629       1 qemu.go:189] 1.19
+I0412 15:30:04.208937       1 qemu.go:189] 2.38
+I0412 15:30:04.265587       1 qemu.go:189] 3.57
+...
+I0412 15:30:09.946197       1 qemu.go:189] 99.21
```

Comment:
I would include the last part of the logs as well.  In my case it was `I0603` and `W0603` instead of `I0412` and `W0412`:

```
I0603 12:08:56.128877       1 prlimit.go:107] ExecWithLimits qemu-img, [info --output=json /data/disk.img]
W0603 12:08:56.137553       1 dataStream.go:343] Available space less than requested size, resizing image to available space 2902560768.
I0603 12:08:56.137921       1 dataStream.go:349] Expanding image size to: 2902560768
I0603 12:08:56.137962       1 prlimit.go:107] ExecWithLimits qemu-img, [resize -f raw /data/disk.img 2902560768]
I0603 12:08:56.158230       1 importer.go:89] import complete
```

Reply:
Added full output for the importer pod.